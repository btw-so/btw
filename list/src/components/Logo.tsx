import React from 'react';
import SVG from 'react-inlinesvg';
import styled from 'styled-components';

export const Wrapper = styled.div`
  align-items: flex-start;
  display: inline-flex;
  font-size: 0;

  svg {
    height: 4.2rem;
    max-height: 100%;
    width: auto;
  }
`;

function Logo() {
  return (
    <Wrapper>
      <SVG src={`${process.env.PUBLIC_URL}/media/brand/icon.svg`} />
    </Wrapper>
  );
}

export default Logo;
