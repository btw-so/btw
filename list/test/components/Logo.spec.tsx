import React from 'react';

import Logo from 'components/Logo';

import { render } from 'test-utils';

describe('Logo', () => {
  it('should render properly', () => {
    const { container } = render(<Logo />);

    expect(container).toMatchSnapshot();
  });
});
