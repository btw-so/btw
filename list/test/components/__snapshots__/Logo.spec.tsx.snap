// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo should render properly 1`] = `
.c0 {
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  font-size: 0;
}

.c0 svg {
  height: 4.2rem;
  max-height: 100%;
  width: auto;
}

<div>
  <div
    class="c0"
  >
    <svg
      src="/media/brand/icon.svg"
    />
  </div>
</div>
`;
